package.json

```diff
@@ -603,70 +593,66 @@
     "value": "#131F37"
   },
   {
-    "name": "pPoloniexMaidsafe",
-    "title": "Poloniex Bitcoin/Maidsafe (\u0243/MAID) ticker?",
+    "name": "pKrakenEther",
+    "title": "Kraken Bitcoin/Ether (\u0243/Ether) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexMaidsafeColor",
-    "title": "Poloniex Bitcoin/Maidsafe Ticker color",
+    "name": "pKrakenEtherColor",
+    "title": "Kraken Bitcoin/Ether Ticker color",
     "type": "color",
-    "value": "#000000"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexBitcoindark",
-    "title": "Poloniex Bitcoin/BitcoinDark (\u0243/BTCD) ticker?",
+    "name": "pKrakenEtherEUR",
+    "title": "Kraken EUR/Ether (\u20ac/ETH) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexBitcoindarkColor",
-    "title": "Poloniex Bitcoin/BitcoinDark Ticker color",
+    "name": "pKrakenEtherEURColor",
+    "title": "Kraken EUR/Ether Ticker color",
     "type": "color",
-    "value": "#000000"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexMonero",
-    "title": "Poloniex Bitcoin/Monero (\u0243/XMR) ticker?",
+    "name": "pKrakenEtherUSD",
+    "title": "Kraken USD/Ether ($/ETH) ticker?",
     "type": "bool",
     "value": false
   },{
-    "name": "pPoloniexMoneroColor",
-    "title": "Poloniex Bitcoin/Monero Ticker color",
+    "name": "pKrakenEtherUSDColor",
+    "title": "Kraken USD/Ether Ticker color",
     "type": "color",
-    "value": "#413ADF"
+    "value": "#FFF"
   },
   {
-    "name": "pPoloniexMoneroUSD",
-    "title": "Poloniex USD/Monero ($/XMR) ticker?",
+    "name": "pPoloniexBelaCoin",
+    "title": "Poloniex Bitcoin/Belacoin (\u0243/BELA) ticker?",
+    "description": "Currencies from Poloniex Exchange",
     "type": "bool",
     "value": false
-  },{
-    "name": "pPoloniexMoneroUSDColor",
-    "title": "Poloniex USD/Monero Ticker color",
-    "type": "color",
-    "value": "#413ADF"
   },
   {
-    "name": "pPoloniexDash",
-    "title": "Poloniex Bitcoin/Dash (\u0243/DASH) ticker?",
+    "name": "pPoloniexBitshares",
+    "title": "Poloniex Bitcoin/ (\u0243/BTS) ticker?",
```

Comment:
The original title was `Poloniex Bitcoin/Bitshares (\u0243/BTS) ticker?` it seems the `Bitshares` name was deleted by accident